www/_posts/2021-12-21-cordova-cli-11.0.0-release.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova CLI 11.0.0 Released!"
+categories: news
+tags: news releases
+---
+
+We are happy to announce that we have just released a major update to our Cordova CLI!
+
+* [cordova@11.0.0](https://www.npmjs.com/package/cordova)
+
+In this CLI release, it will also be using the latest internal libraries and template:
+
+* [cordova-lib@11.0.0](https://www.npmjs.com/package/cordova-lib)
+* [cordova-create@4.0.0](https://www.npmjs.com/package/cordova-create)
+* [cordova-app-hello-world@6.0.0](https://www.npmjs.com/package/cordova-app-hello-world)
+
+## Release Highlights
+
+In this release, we have dropped Nodejs 10 support. The minimum supported version which Cordova requires is 12.x.
+
+In all releases, we have updated all npm packages to the possible latest release in which Cordova can support.
+
+**Cordova CLI 11.0.0:**
+
+In this release, we fixed an issue where users were unable to use Cordova CLI after upgrading their macOS environment to macOS Monterey. This issue was caused by one of the package dependencies, `insight`. The previous workaround was to reinstall Cordova CLI, but now this is no longer required with this release.
+
+**Cordova Lib 11.0.0:**
+
+We have bumped all platform pinnings to use the latest released platforms.
+
+* `cordova-android@^10.1.1`
+* `cordova-electron@^3.0.0`
+* `cordova-ios@^6.2.0`
+
+**Cordova Create 4.0.0:**
+
+In this release, all new projects will start with the newest App Hello World template.
+
+**Cordova App Hello World Template 6.0.0:**
+
+Please refer to the [App Hello World Release Blog](https://cordova.apache.org/news/2021/10/31/template-release.html) post for more details.
+
```

Comment:
```suggestion

## Greetings

The Apache Cordova team wishes you Happy Holidays and a good start for the new year.

```

Should we include some greetings like this? As this release is just in time and a good opportunity?